iModelCore/ECPresentation/Source/Content/ContentDescriptorBuilder.cpp

```diff
@@ -430,16 +432,21 @@ struct ContentPropertiesAppender : ContentSpecificationsHandler::PropertyAppende
             propertyClassAlias = pathFromPropertyClassToNavigationPropertyTargetClass.GetTargetClass().GetAlias().c_str();
             DIAGNOSTICS_DEV_LOG(DiagnosticsCategory::Content, LOG_TRACE, Utf8PrintfString("Switched to alias `%s` as it's a navigation property.", propertyClassAlias));
             }
-        result.MergeWith(_DoAppendProperty(ecProperty, propertyClassAlias, overrides));
+        result.MergeWith(_DoAppendProperty(ecProperty, propertyClass, propertyClassAlias, overrides));
         return result;
         }
 
+    ContentSpecificationsHandler::PropertyAppendResult _AppendCalculatedProperty(ECClassCP ecClass, CalculatedPropertiesSpecificationCP spec, Utf8StringCR label, Utf8StringCR name) override
+        {
+        return _DoAppendCalculatedProperty(ecClass, spec, label, name);

```

Comment:
what's the purpose of `_DoAppendCalculatedProperty`?

Reply:
Just followed the appender's pattern. I can remove if you want.

Reply:
`_DoAppendProperty` is required, because we need `_Append` to some additional stuff before subclasses can take over - that's where `_DoAppendProperty` comes in. In case of `_AppendCalculatedProperty`, there's nothing additional being done, so there's no need to an additional virtual function.

Reply:
fixed